railties/test/generators/named_base_test.rb

```diff
@@ -80,10 +80,13 @@ def test_scaffold_plural_names_as_ruby
 
   def test_application_name
     g = generator ['Admin::Foo']
-    Rails.stubs(:application).returns(Object.new)
```

Comment:
I think this doesn't bring the default implementation back making our tests order dependent